### DISCUSSION TITLE
Events are reported as "by Supervisor" but should they be "by Node-RED"?

### DISCUSSION THREAD

**Discussion by eprouty** [hassio-addons/addon-node-red#1353] [Q&A] [answered]
Noticed that when I am looking at the history of an entity or service call, actions that Node-RED has taken show up as "by Supervisor":
![image](https://user-images.githubusercontent.com/5383520/170906857-e35a3979-9da6-4191-a373-ddbb0619f8f0.png)

This is a bit unintuitive and would be a lot cleaner to track and understand if it said "by Node-RED". 

Thoughts on if this change should happen? and would it be easy to implement?

**Comment by frenck** [comment, marked as answer]
> This is a bit unintuitive and would be a lot cleaner to track and understand if it said "by Node-RED".

The communication is done by the Supervisor. That is why you see that.

> Thoughts on if this change should happen?

That is up to you. You can create a dedicated user for Node-RED and configure Node-RED to use those credentials instead.
